opsdroid/events.py

```diff
@@ -269,12 +311,14 @@ def __init__(
         self._mimetype = mimetype
         self._file_bytes = file_bytes
         self.url = url
+        self._url_headers = url_headers
 
     async def get_file_bytes(self):
         """Return the bytes representation of this file."""
         if not self._file_bytes and self.url:
             async with aiohttp.ClientSession() as session:
-                async with session.get(self.url) as resp:
+                _LOGGER.debug(self._url_headers)
```

Comment:
I assume we are debug logging the headers here for a reason. Perhaps it would be good to include the url too for some context?

Reply:
I am not sure we are including this for any real reason here...